src/pages/ServerRoleSummary/UpdateNetworks.js

```diff
@@ -132,6 +190,71 @@ class UpdateNetworks extends Component {
       .sort(alphabetically);
   }
 
+  removeAddress = (idx) => {
+    this.setState(prev => {
+      let addresses = prev.addresses.slice();
+      addresses.splice(idx, 1);
+      return {addresses: addresses};
+    });
+    if(!this.state.isFormChanged) {
```

Comment:
Instead of using a separate if, just put the ``isFormChanged: true`` in the structure returned on line 197.

Reply:
ok. will update